### PR TITLE
fix(collapsible): display necessary part of title

### PR DESCRIPTION
**Pull request by barmac**
https://user-images.githubusercontent.com/28307541/112017124-9e445680-8b2d-11eb-865d-b640e3ee04e1.mp4

Related to https://github.com/camunda/camunda-modeler/issues/2166

### CODE REVIEW

File: styles/properties.less [hunk 1]
```diff
@@ -112,7 +118,7 @@
     flex-direction: row;
     align-items: center;
 
-    > .bpp-error-message {
+    >.bpp-error-message {
```
Comment (nikku) [inline styles/properties.less:121]: What and why do we reformat the whole file?

Comment (barmac) [reply styles/properties.less:121]: lol I committed only what I wanted to remove 🙈

Comment (barmac) [reply styles/properties.less]: It should be OK now. Thanks for the review!

**Comment by barmac** [comment]
I've removed the unintended formatting and added a small cleanup on top.

**Review by nikku — APPROVED** [review]

**Pull request merged**